ext/tags_test.go

```diff
@@ -1,20 +1,27 @@
 package ext_test
 
 import (
+	"reflect"
 	"testing"
 
+	"github.com/opentracing/basictracer-go"
+	"github.com/opentracing/basictracer-go/testutils"
```

Comment:
Should be ok, it's only a test dependency. If it becomes an issue, someone will have to put a trivial tracer impl that supports attributes here.